examples/python/host/managed/after_create/12-ppc64-wrapper.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
```

Comment:
This is out of scope for foreman_hooks, I'm not able to review or maintain it - please publish it to a different repository.